src/screens/PostScreen.tsx

```diff
@@ -7,37 +7,65 @@ import { styles } from './styles';
-        experiencing homelessness. The organization, founded in 2015, is
-        dedicated to addressing both immediate needs and underlying causes of
-        homelessness through a combination of shelter services, job training
-        programs, and mental health support. read more online:
-        https://www.codetenderloin.org/
-      </Text>
-      <Text>
-        Image URL:
-        'https://cdn.britannica.com/51/178051-050-3B786A55/San-Francisco.jpg'
-      </Text>
-      <HeartIcon />
-      <Text>256 Likes</Text>
-      <ShareIcon />
-      <ProfilePlaceholder />
-      <Text>philip_ye</Text>
-      <Text>September 20</Text>
-      <Text>
-        This organization is doing amazing work tackling the complex root causes
-        of the issue.
-      </Text>
-      <ProfilePlaceholder />
-      <Text>vppraggie</Text>
-      <Text>September 21</Text>
-      <Text>Thanks for sharing!</Text>
+      <View style={styles.post}>
+        <View style={styles.person_header}>
+          <View style={styles.name_profile}>
+            <ProfilePlaceholder />
+            <Text style={styles.username}>rbeggs</Text>
+          </View>
+          <Text style={styles.post_date}>September 19</Text>
+        </View>
+        <Text style={styles.da_text}>
+          In response to the growing homelessness crisis in San Francisco, a
+          local nonprofit organization, Code Tenderloin, has launched a
+          comprehensive initiative aimed at providing long-term solutions for
+          individuals experiencing homelessness. The organization, founded in
+          2015, is dedicated to addressing both immediate needs and underlying
+          causes of homelessness through a combination of shelter services, job
+          training programs, and mental health support. read more online:
+          <Text style={styles.da_link}>https://www.codetenderloin.org/</Text>
+        </Text>
+        <Image
+          style={styles.da_image}
+          source={{
+            uri: 'https://cdn.britannica.com/51/178051-050-3B786A55/San-Francisco.jpg',
+          }}
+        />
+        {/* <Text>
+          Image URL:
+          'https://cdn.britannica.com/51/178051-050-3B786A55/San-Francisco.jpg'
+        </Text> */}
+        <View style={styles.interactions}>
+          <View style={styles.likes}>
+            <HeartIcon />
+            <Text style={styles.username}>256 Likes</Text>
+          </View>
+          <ShareIcon />
+        </View>
+      </View>
+
+      <View style={styles.comments}>
+        <View style={styles.person_header}>
+          <View style={styles.name_profile}>
+            <ProfilePlaceholder />
+            <Text style={styles.username}>daviddd</Text>
+          </View>
+          <Text style={styles.post_date}>September 20</Text>
+        </View>
+        <Text style={styles.c_text}>
+          This organization is doing amazing work tackling the complex root
+          causes of the issue.
+        </Text>
```

Comment:
Let's create a separate react component for the Comments to reduce rewriting code.